Tringo.AirportsParser/Program.cs

```diff
@@ -0,0 +1,129 @@
+﻿using HtmlAgilityPack;
+using Newtonsoft.Json;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net;
+
+namespace Tringo.AirportsParser
+{
+    class Program
+    {
+        private const string WIkiAirportsAddress = "https://en.wikipedia.org/wiki/List_of_international_airports_by_country";
+        private const string outputFileName = "AirportsPassengers.json";
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
```

Comment:
What is this "Hello World" in the production code?

Reply:
Also, if you want to debug, don't use console, use logger

Reply:
It's one time running code, who cares. I'd easier drop it rather than refactor ;)

Reply:
I already added generated file, so it's just for reporting